### DISCUSSION TITLE
08.02 팀미팅

### DISCUSSION THREAD

**Discussion by NariP** [wanted-preonboarding-TEAM2/JARANDA#4] [Scrum]
08.02 팀미팅

**Comment by ha3158987** [comment]
- 관리자 페이지 와이어프레임 작성
- feature list 간단히 작성 
- 기능별로 분업
    - data table : 시형님, 박현아
    - search bar : 나리님, 상혁님

**Comment by simian114** [comment]
- 관리자 페이지 searchBox UI 작성
  - 박나리, 남상혁
